CMakeLists.txt

```diff
@@ -200,7 +202,11 @@ if(NOT COMMAND find_host_program)
 endif()
 
 # Tests require Python3
-find_host_package(PythonInterp 3 REQUIRED)
+if(NOT SPIRV_SKIP_TESTS)
+  find_host_package(PythonInterp 3 REQUIRED)
+else()
+  find_host_package(PythonInterp 2.7 REQUIRED)
```

Comment:
Should we just skip finding Python if not building the tests?

Reply:
We're still using it for generating files

Reply:
Ok